etc/setup/scripts/subtasks/tyk_setup.sh

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env bash
+# This script will set up a full tyk environment on your local CanDIGv2 cluster
+
+# ** Be sure to invoke this from the Makefile at the project's root directory [CanDigV2] **
+
+# Load Tyk template (.tpl) files, populate them 
+# with project .env variables, and then spit 
+# them out to ./lib/tyk/volumes/*
+
+mkdir -p ${PWD}/lib/tyk/volumes
+
+# api_auth.json
+echo "Working on api_auth.json .."
+envsubst < ${PWD}/etc/setup/templates/configs/tyk/confs/api_auth.json.tpl > ${PWD}/lib/tyk/volumes/api_auth.json
```

Comment:
these templates can be embedded in lib/ or under etc/tpl to generalize templates

Reply:
will consider this in another branch